.github/dependabot.yml

```diff
@@ -9,3 +9,5 @@ updates:
     schedule:
       interval: "weekly"
     target-branch: "9.x"
+    commit-message:
+      prefix: "[9.x]"
```

Comment:
@ndeverge Nice! I haven't thought of that.

However, after reading the [`commit-message` docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message) the commit message will end up as
```
# see the colon?
[9.x]: https://github.com/playframework/play-silhouette/pull/116
```
I think you do not want a colon however? (That's how we do it with scala-steward in all our Play repos)
```
[9.x] https://github.com/playframework/play-silhouette/pull/116
```

> When you specify a prefix for commit messages, GitHub will **automatically add a colon between the defined prefix and the commit message provided the defined prefix ends with a letter, number, closing parenthesis, or closing bracket**.

Further:
> This means that, for example, if you end the prefix with a whitespace, there will be no colon added between the prefix and the commit message. The code snippet below provides examples of both in the same configuration file.

So I think we want to change this to
```yml
    commit-message:
      prefix: "[9.x] " # <- see the whitespace after ]
```

What do you think?

Reply:
Ah sorry, just realized the PR is from @MathisGuillet1... 
Please see
- #120